connectors/src/lib/data_sources.ts

```diff
@@ -770,6 +771,7 @@ export async function upsertTableFromCsv({
     `/data_sources/${dataSourceConfig.dataSourceId}/tables/csv`;
   const dustRequestPayload: UpsertTableFromCsvRequestType = {
     name: tableName,
+    parentId: null,
```

Comment:
do we send null even if parents[1] is set?

Reply:
done